pyvcloud/vcd/vdc.py

```diff
@@ -373,6 +373,42 @@ def list_edge_gateways(self):
                 })
         return edge_gateways
 
+    def get_nat_rules(self, edge_gateway_name):
+        """Request the nat rules of an edge gateway.
+
+        :param edge_gateway_name: (str): The name of the edge gateway.
+
+        :return: A list of :class:`lxml.objectify.StringElement' objects
+            representing the nat rules.
+
+        :raises: Exception: if there is no edge gateway with the provided name.
+        :raises: Exception: if there are no nat rules on the edge gateway
+            with the provided name.
+        """
+        edge_gateways = self.list_edge_gateways()
+        nat_rules = []
+        for edge_gateway in edge_gateways:
+            if edge_gateway_name == edge_gateway.get('name'):
+                edge_gateway_full = \
+                    self.client.get_resource(edge_gateway.get('href'))
+                if hasattr(edge_gateway_full, 'Configuration') and \
+                   hasattr(edge_gateway_full.Configuration,
+                           'EdgeGatewayServiceConfiguration') and \
+                   hasattr(edge_gateway_full.Configuration.
+                           EdgeGatewayServiceConfiguration,
+                           'NatService'):
+                    for nat_rule in edge_gateway_full.Configuration.\
+                            EdgeGatewayServiceConfiguration.NatService.NatRule:
+                        nat_rules.append(nat_rule)
+                if len(nat_rules) > 0:
+                    return nat_rules
+                else:
```

Comment:
Please see my comment, regarding throwing exceptions, at the top of this function.